src/Simplify.elm

```diff
@@ -4108,6 +4135,54 @@ isNegatableOperator op =
             Nothing
 
 
+basicsToFloatChecks : CheckInfo -> Maybe (Error {})
+basicsToFloatChecks checkInfo =
+    case Evaluate.getNumber checkInfo checkInfo.firstArg of
```

Comment:
You'll have to use `getInt` because `getNumber` also parses `1.2` and inferred as `1.2` for example.
If you think the naming is off and should be `getFloat`, feel free to change it (along with AstHelpers.getUncomputedNumber).

Tests for this would also be nice.

Reply:
Nah, the naming is fine. I copy-pasted from somewhere thinking I should change it to `getInt` and then somehow forgot. I'll add a test.